lib/index.js

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from "react";
+import { useStoryblokBridge as useSbBridge } from "@storyblok/react";
```

Comment:
Do we need to import useStoryblokBridge with an alias?
useStoryblokBridge is already being imported/exported in the line below. Can we use that same instance of the function?

Reply:
I believe yes as it shows as not defined if I don't use the alias as in this screenshot.

```javascript
// what I tested

import { useEffect, useState } from "react";
// 👇 excluded the alias
// import { useStoryblokBridge as useSbBridge } from "@storyblok/react";
export { useStoryblokBridge, storyblokInit, apiPlugin, StoryblokComponent, storyblokEditable } from "@storyblok/react";

export function useStoryblokState(originalStory, bridgeOptions) {
  if (typeof originalStory.content === "string") originalStory.content = JSON.parse(originalStory.content);

  let [story, setStory] = useState(originalStory);
  useEffect(() => {
    // 👇 changed from alias to useStoryblokBridge
    useStoryblokBridge(story.internalId, (newStory) => setStory(newStory), bridgeOptions);
  }, []);

  return story;
}
```

<img width="1019" alt="Screenshot 2022-06-05 at 15 54 47" src="https://user-images.githubusercontent.com/25793187/172054206-732f9e9d-02ad-4ca2-95a6-736ad9ec00ba.png">

Reply:
Yes. The line below is `export from` so you still need to import it if you want to use it

Reply:
Oooh that's interesting. Thanks for the insights @schabibi1 @alexjoverm